### DISCUSSION TITLE
how to create _Layout.cshtml for modules?

### DISCUSSION THREAD

**Discussion by MRSheikh** [OrchardCMS/OrchardCore#9825] [Q&A] [answered]
I have some modules for Orchardcore cms. is there a way to implement `_Layout.cshtml ` for each module?

**Comment by Skrypt** [comment, marked as answer]
You can see an example here : 

https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Modules/OrchardCore.ContentPreview/Views/Preview/Index.cshtml#L2

I believe that it works in every modules already.
